tutorials/algorithms/11_gradients_framework.ipynb

```diff
@@ -0,0 +1,718 @@
+{
```

Comment:
<p>This explanation is for the "Using Estimator" subsection. So I would add the title before the "Given a hermitian operator...". And maybe, remove the last sentence, we can give a bit more detail on probability gradients on the following subsection.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1463/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<blockquote>we can give a bit more detail on probability gradients on the following subsection.</blockquote><p>What details should be added?</p><p> </p>

Reply:
<p>I meant that the sentence on probability gradients could be moved to the sampler section. The way it currently looks like is good. </p>